app/models/trend.rb

```diff
@@ -13,6 +13,10 @@ def to_param
     Parameterizable.parameterize("#{id}-#{to_label}")
   end
 
+  def self.title_name
```

Comment:
Why do we need this?

Reply:
when we have another attribute which reflects the name,  we should add `title_name` class method
as you can see [here](https://github.com/theforeman/foreman/blob/68388bc261fd72796d9e2af73ed092257d69fb5b/app/services/breadcrumbs_options.rb#L55)

Reply:
Got it, thanks.